materialrangebar/src/main/res/values/attrs.xml

```diff
@@ -6,6 +6,7 @@
         <attr name="mrb_tickInterval" format="float"/>
         <attr name="mrb_tickHeight" format="dimension"/>
         <attr name="mrb_tickColor" format="color"/>
```

Comment:
i would suggest you rename `mrb_tickColor` to `mrb_defaultTickColor` as this would be used incase user doesn't set multiple `mrb_tickColors`

Reply:
Done. Note developers upgrading version will have to change to the new name.